test/expected/plan_expand_hypertable-11.out

```diff
@@ -2475,6 +2475,75 @@ SELECT time FROM regular_timestamptz UNION ALL SELECT time FROM metrics_timestam
  Tue Feb 01 00:00:00 2000 PST
 (128 rows)
 
+-- test nested join qual propagation
+:PREFIX
+SELECT * FROM (
+SELECT m1.time FROM metrics_timestamptz m1 FULL OUTER JOIN metrics_timestamptz_2 m2 ON true WHERE m2.time = m1.time AND m1.device_id = m2.device_id AND m2.time < '2000-01-10' AND m1.device_id = 1
```

Comment:
Good to test some more variants here:
SELECT * FROM (q1 outer join q2 ) , 
1) with q1 as select * from m1 **left outer join** m2 ON  **<non -trivial clause >** here.
2) with q1 as select * from m1 **inner join** m2 ON **<non-trivial clause>**